src/sm/sm_main/debug/full-tracer.js

```diff
@@ -8,7 +8,7 @@ const opDecContext = ['SELFDESTRUCT', 'STOP', 'RETURN'];
 const { Scalar } = require("ffjavascript");
```

Comment:
Add comments with structure as they are in `full-tracer-utils.js` for this file s well.
It will help external consumers to understand the code

Reply:
Agree